source/Serial/NetworkSerial.cpp

```diff
@@ -155,20 +155,20 @@ NetworkSerial::flush(
     create_task( _current_store_operation )
```

Comment:
that is a good point. previously, we used this store operation to tell if any of the previous store operations failed, and I remember we also had issues with items being prematurely destructed if we didn't store references to them. Now that we are using the store operation immediately, a capture is better.